### DISCUSSION TITLE
v1.0.0-beta.4

### DISCUSSION THREAD

**Discussion by kamiya4047** [ExpTechTW/rts-map#12] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/ExpTechTW/rts-map/releases/tag/v1.0.0-beta.4'>v1.0.0-beta.4</a>.</em>

**Comment by liug1086** [comment]
我的RTS MAP無法登入要怎麼辦呢，希望可以解決。
![螢幕擷取畫面 2024-12-07 130335](https://github.com/user-attachments/assets/b986c325-b357-4b7a-9131-7fd65c37cfe0)

**Reply by aalove1130** [reply]
一樣
